File: java/URI_1072.java
package URI;
import java.util.Scanner;

public class URI_1072 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		int i = 0;
		int dentro = 0;
		int fora = 0;
		
		while(i < n) {
			int num = sc.nextInt();
			if(num < 10 || num > 20) {
				fora += 1;
			}
			else {
				dentro += 1;
			}
			i++;
		}
		
		System.out.println(dentro + " in");
		System.out.println(fora + " out");		
		
		sc.close();		

	}

}

File: kotlin/URI_1072.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    val n = sc.nextInt()
    var dentro = 0
    var fora = 0
    for (i in 1..n) {
        val numero = sc.nextInt()
        when (numero) {
            in 10..20 -> dentro++
            else -> fora++
        }
    }

    println("$dentro in")
    println("$fora out")
}
